eng/common/scripts/job-matrix/Create-PrJobMatrix.ps1

```diff
@@ -40,12 +40,13 @@ param (
   [Parameter(Mandatory = $true)][string] $PackagePropertiesFolder,
   [Parameter(Mandatory = $true)][string] $PRMatrixFile,
   [Parameter(Mandatory = $true)][string] $PRMatrixSetting,
-  [Parameter(Mandatory = $False)][string] $DisplayNameFilter,
-  [Parameter(Mandatory = $False)][array] $Filters,
-  [Parameter(Mandatory = $False)][array] $IndirectFilters,
-  [Parameter(Mandatory = $False)][array] $Replace,
-  [Parameter(Mandatory = $False)][bool] $SparseIndirect = $true,
-  [Parameter(Mandatory = $False)][int] $PackagesPerPRJob = 10,
+  [Parameter(Mandatory = $false)][string] $DisplayNameFilter,
```

Comment:
I saw the differing cases and

![sdf](https://media2.giphy.com/media/xT8qB7GQDfnldWTFhS/giphy.gif)